src/scripts/modules/media/body/content-baked.less

```diff
@@ -250,6 +250,17 @@ h1.example-title .text {
   }
 }
 
+[data-type="exercise"] {
+  [data-type="problem"] {
+    p {
```

Comment:
Should we add something to specify that only the first paragraph follows the number inline?

Reply:
Good idea, just made another commit @tomjw64